compiler/kernel/op.go

```diff
@@ -325,7 +416,21 @@ func (b *Builder) compileSequential(seq *dag.Sequential, parents []zbuf.Puller)
 	return parents, nil
 }
 
-func (b *Builder) compileParallel(par *dag.Parallel, parents []zbuf.Puller) ([]zbuf.Puller, error) {
+func (b *Builder) compileParallel(parallel *dag.Parallel, parents []zbuf.Puller) ([]zbuf.Puller, error) {
```

Comment:
Nit: Stick with par since that's what we call a *dag.Parallel everywhere else.
```suggestion
func (b *Builder) compileParallel(par *dag.Parallel, parents []zbuf.Puller) ([]zbuf.Puller, error) {
```